Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
     nio4r (2.5.8)
-    nokogiri (1.13.3-x86_64-darwin)
+    nokogiri (1.13.8-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
     pagy (5.10.1)
       activesupport
     parallel (1.22.1)
-    parser (3.1.1.0)
+    parser (3.1.2.1)
       ast (~> 2.4.1)
-    pg (1.3.5)
+    pg (1.4.3)
     propshaft (0.6.4)
       actionpack (>= 7.0.0)
       activesupport (>= 7.0.0)
       rack
       railties (>= 7.0.0)
-    public_suffix (4.0.6)
-    puma (5.6.4)
+    public_suffix (5.0.0)
```

Comment:
### [public_suffix](https://simonecarletti.com/code/publicsuffix-ruby)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 4.0.6 → 5.0.0

[_[change-log](https://github.com/weppos/publicsuffix-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/weppos/publicsuffix-ruby/tree/v5.0.0), [gem-diff](https://my.diffend.io/gems/public_suffix/4.0.6/5.0.0)_]





<details>
<summary>Commits</summary>

A change of **37** commits. See the full changes on [the compare page](https://github.com/weppos/publicsuffix-ruby/compare/v4.0.6...v5.0.0).

These are the first 10 commits:
- (0898e5a) [Add definitions section to README.md (#176)](https://github.com/weppos/publicsuffix-ruby/commit/0898e5a48ad0b1c604c2cc14bfe9a393dce180ad)
- (ba72b15) [Disable Lint/ConstantDefinitionInBlock in specs](https://github.com/weppos/publicsuffix-ruby/commit/ba72b15aca34e3444eaf9a2341d9176d883c02b9)
- (6fa9751) [Disable Lint/ConstantDefinitionInBlock in Rakefile](https://github.com/weppos/publicsuffix-ruby/commit/6fa975174410f64942559026d027621d1b837753)
- (d6c251a) [Create Dependabot config file (#178)](https://github.com/weppos/publicsuffix-ruby/commit/d6c251aad4f6daf0d41dea9d921b1faa1bdb8849)
- (c4c3012) [Use instance_of? instead of comparing classes](https://github.com/weppos/publicsuffix-ruby/commit/c4c301231549f98b53bd987c9398b3a366aad815)
- (db76aea) [Disable Coverage](https://github.com/weppos/publicsuffix-ruby/commit/db76aea35c5e10f4ec64f082f5b5b48d79210f65)
- (f03cec8) [Test with Ruby 3](https://github.com/weppos/publicsuffix-ruby/commit/f03cec89b2cefbaa644c4a76545b04a40af30187)
- (c50a052) [CI: Remove EOL versions](https://github.com/weppos/publicsuffix-ruby/commit/c50a052e89604a28a09ac4194f70957bf02db486)
- (47bee57) [Format](https://github.com/weppos/publicsuffix-ruby/commit/47bee57a1bc4b0834dd92dc72fe7690ab2af63c0)
- (450825e) [Replace Travis with GitHub Actions](https://github.com/weppos/publicsuffix-ruby/commit/450825e80c579b70ac423206e57c13384051a31f)


</details>